File: _examples/csharp/tuples.cs
// C# 值元组 (C# 7.0+)
var person = (25, "Alice", 1.68);
(int age, string name, double height) = person;

// 命名元组
var named = (Age: 25, Name: "Alice", Height: 1.68);
Console.WriteLine(named.Name);

// 元组方法返回
(int min, int max) GetRange() => (0, 100);

File: _examples/swift/tuples.swift
// Swift 元组
let person: (Int, String, Double) = (25, "Alice", 1.68)

// 访问元素
let age = person.0
let name = person.1

// 命名元组
let named = (age: 25, name: "Alice", height: 1.68)
print(named.name)

// 解构
let (a, n, h) = person